### ISSUE TITLE
Debugging a Go contract on macOS requires Go 1.12

### ISSUE THREAD

**Issue by liam-grace** [IBM-Blockchain/blockchain-vscode-extension#781]
## Description
Debugging a Go contract with Go version 1.11 (on macOS) gives the error:
```
could not launch process: decoding dwarf section info at offset 0x0: too short
```
This issue disappears when Go is updated to 1.12
(Only macOS tested)

## Expected Behavior
User should be aware of the Go version required to debug contracts

## Actual Behavior
No mention in the README

## Possible Fix
Add Go version to the readme

## Your Environment
* IBM Blockchain Platform Extension Version: 0.3.3
* Visual Studio Code version: 1.32.3
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1): Go 1.11
* Operating System and version: macOS 10.14.3

**Comment by davidkel** [comment]
I've tried to debug Go in linux using `go version go1.11.4 linux/amd64`
I was expecting it to fail as per this issue but I get a different message so the error may vary from platform to platform. Adding it here in case someone else searches for it.
On ubuntu 18.04 I get the following message
```
API server listening at: 127.0.0.1:49048
could not launch process: could not get .debug_frame section: could not find .debug_frame section
```
This was fixed by moving to go 1.12.1

**Comment by cazfletch** [comment]
We should update the readme to say that you need at least go 1.12.1
